.github/workflows/ci.yml

```diff
@@ -8,9 +8,9 @@ on:
 
 # Configure GitHub Actions cancel in progress workflow to avoid redundant runs in pull requests.
 # See: https://docs.github.com/en/enterprise-cloud@latest/actions/writing-workflows/choosing-what-your-workflow-does/control-the-concurrency-of-workflows-and-jobs
-concurrency:
+  concurrency:
```

Comment:
Maybe this is the problem with `on`? The indentation makes this belong to the `on` from line 3?

Reply:
It is. Thanks!